web/content/js/versions.js

```diff
@@ -212,7 +220,11 @@ const navVersions = [
             "path": "Installing_Proxy"
           },
           {
-            "title": "Upgrading Foreman",
+            "title": "Updating Katello",
+            "path": "Updating_Project"
+          },
+          {
+            "title": "Upgrading Katello to nightly",
```

Comment:
And then this is
```suggestion
            "title": "Upgrading Katello to 3.7",
```

Reply:
Are you sure version `3.7` is also applicable to Katello alongside `Foreman`? I guess it should be `4.9`

But, my query is why do we write `3.7` when we are already there? It should be `nightly` or higher version IMHO.

Reply:
You're right, 4.9 since it's Katello. But it's 4.9 since this is the Foreman 3.7 set of links.

Reply:
I hope I understood correctly.